flowmachine/Pipfile

```diff
@@ -37,6 +40,10 @@ asynctest = "*"
 cachey = "*"
 approvaltests = "*"
 watchdog = "*"
+ipdb = "*"
 
 [requires]
 python_version = "3.7"
+
+[pipenv]
+allow_prereleases = true
```

Comment:
Need to be careful here, because this is globally applicable.

Reply:
Yes, I was slightly concerned about this too, but I didn't think pipenv would allow us to install a pre-release version of marshmallow without specifying this flag. But actually when I tried it again with the pinned version it seems to work, so maybe this can be removed? I'll leave it in for now but may try another CI run with it removed once we have addressed everything else.